src/patterns/OcFileActions.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div uk-offcanvas="mode: slide" class="uk-offcanvas uk-offcanvas-bottom oc-file-actions">
+    <div class="uk-offcanvas-bar">
+      <span v-text="_label"></span>
+      <ul class="uk-nav">
+        <template v-for="(item, i) in items">
+          <li uk-toggle="target:.oc-file-actions" :key="i" @click="item.onClick">
```

Comment:
@DeepDiver1975 @tomneedham is this the right way to hand over a handler to the vue model?

Reply:
looks good to me and works ... so :+1: